docs/application/dotnet/samples/telephony/overview.md

```diff
@@ -37,7 +37,11 @@ for TD:
 <table>
 	<tbody>
 		<tr>
-			<td>There is no sample.</td>
+			<td><img alt="" height="180" src="media/w59telephony.png" width="180"/></td>
+			<td>
+			<p><strong>(W) Telephony</strong> [In progress]</p>
+			<p>This sample application demonstrates how to work with Telephony API in order to get the call, modem, network, and sim information from the telephony framework.</p>
```

Comment:
@Reni-90 I have a question to this update.

For the mobile sample in this page we have "This sample application demonstrates how to work with the Telephony API to receive call, modem, network, and sim information from the telephony framework."

This sentence is reviewed by @Reni-90 on https://github.com/Samsung/tizen-docs/pull/368,
and I think it would be better to keep the similar sentence for similar items.
How about using sentences with consistency for each Telephony sample apps for Mobile and Wearable?

Reply:
@mijin85cho 
I completely agree with you.
Even though both sentences convey the same meaning. For consistency, let us keep the same sentences.
Thank you.

@sgchoi5 
Please keep the same sentence to maintain consistency:
 "This sample application demonstrates how to work with the Telephony API to receive the call, modem, network, and sim information from the telephony framework."

Reply:
Fixed.